docs/command_line_reference.rst

```diff
@@ -674,7 +674,7 @@ Enable it with ``use_ssl:true``.
 **TLS/SSL Certificate Verification**
 
 Server certificate verification is controlled with the ``verify_certs`` boolean. The default value is `true`. To disable use ``verify_certs:false``.
-If ``verify_certs:true``, Rally will attempt to verify the certificate provided by Elasticsearch. If they are private certificates, you will also need to supply the private CA certificate using ``ca_certs:'/path/to/cacert.pem'``.
+If ``verify_certs:true``, Rally will attempt to verify the certificate provided by Elasticsearch. If they are private self-signed certificates, you will also need to supply the private self-signed CA certificate in PEM format using ``ca_certs:'/path/to/cacert.pem'``.
```

Comment:
The wording was incorrect, even before this PR. The server (Elastic) does not present a self-signed a certificate, it presents a server certificate that was signed by a root CA. And yes, all root CA are self-signed, but I think "private" is a better description than "self-signed".

```suggestion
If ``verify_certs:true``, Rally will attempt to verify the certificate provided by Elasticsearch. If this certificate is signed by a private certificate authority (CA), you will also need to supply that CA in PEM format using ``ca_certs:'/path/to/cacert.pem'``.
```

You don't have to agree, please merge with or without this change. It's fine to merge a doc change on Friday.

Reply:
I like the revised wording, thank you!